pom.xml

```diff
@@ -140,7 +140,10 @@
         <dependency>
             <groupId>org.testobject</groupId>
             <artifactId>testobject-java-api</artifactId>
-            <version>0.0.26</version>
+            <version>0.0.27</version>
+            <!-- Using local jar for development. Remove when releasing -->
```

Comment:
Haha ;)

Could you give me more information on the mvn install trick (do you have a link or something I could google), that might be really helpful